test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -634,4 +675,9 @@ namespace integration_framework {
       boost::filesystem::remove_all(iroha_instance_->block_store_dir_);
     }
   }
+
+  const std::shared_ptr<IrohaInstance>
+      &IntegrationTestFramework::getIrohaInstance() const {
```

Comment:
`const shared_ptr<..>`? :\ Maybe just `const IrohaInstance&`?

Also I'm not really sure that this component should be publicly available